### PR TITLE
Update the sites content

### PR DESCRIPTION
**Pull request by ghost**
Uppdate the sites images and text according to what the customer wanted.
Also update the Gemfile.lock to include json 1.8.3 instead of 1.8.1 due to crashing when executing bundle install.

### CODE REVIEW

File: .bowerrc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "directory" : "/_sass"
-}
\ No newline at end of file
+  "directory" : "public/components"
+}
```
Comment (scmx) [inline .bowerrc:3]: Changing _sass to public/components looks like an unrelated change. Was there a particular reason why this was changed together with json version?

**Comment by scmx** [comment]
Looking good

File: .bowerrc (hunk 1 above)
Comment (alex-ross) [inline .bowerrc:3]: @scmx: @MrMagnus had some issues with permissions so I told him to add this if there wasn't a file already that defined a directory.

However. As you say this change will break stuff. But the previous one isn't correct. when the value is `/_sass` it wants to create that folder in my computers root directory 😄. I'll will reverse this change and fix the old one.
